models/gamesM.js

```diff
@@ -1,25 +1,131 @@
 const pool = require("../db/connection.js");
-const { checkReviewExists, checkUsernameExists } = require("../db/db.js");
+const {
+  checkReviewExists,
+  checkUsernameExists,
+  checkCategoryExists,
+} = require("../db/db.js");
 
 exports.selectCategories = () => {
   return pool.query("SELECT * FROM categories;").then((result) => {
     return result.rows;
   });
 };
 
-exports.selectReviews = () => {
-  return pool
-    .query(
-      `SELECT title, category, designer, owner, review_img_url, reviews.created_at, reviews.review_id, reviews.votes, CAST(COUNT(comments.comment_id) AS INTEGER) as comment_count FROM reviews
-      LEFT JOIN comments
-      ON reviews.review_id = comments.review_id
-      GROUP BY reviews.review_id
-      ORDER BY reviews.created_at DESC
-      ;`
-    )
-    .then((result) => {
-      return result.rows;
+// exports.selectReviews = (category, sort_by, order) => {
+//   return pool
+//     .query(
+//       `SELECT
+
+//       title,
+//       category,
+//       designer,
+//       owner,
+//       review_img_url,
+//       reviews.created_at,
+//       reviews.review_id,
+//       reviews.votes,
+//       CAST(COUNT(comments.comment_id) AS INTEGER) as comment_count FROM reviews
```

Comment:
if your tests are passing for your new function, forget this old commented one - you've moved onto better things now! Can be deleted.